### DISCUSSION TITLE
A style for Stream Deck Neo Infobar can't show AM/PM in Japanese

### DISCUSSION THREAD

**Discussion by TM0428** [elgatosf/elgato#131] [General] [answered]
I was using Dot Style infobar and noticed that the AM/PM display was missing.
![image](https://github.com/user-attachments/assets/ff22e8d3-2416-41b7-9236-74b1b5f18596)
expected:
02:36 午後(or 午前)
but:
02:36 午
This problem occurs both in the AM/PM.

**Comment by ChekTek** [comment, marked as answer]
Thank you for pointing this out! I have reported the issue to the Stream Deck core team.
